projection/validator_delegation/migrations/20220117130920_create_view_vd_unbonding_delegation_queue.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE view_vd_unbonding_delegation_queue (
+    id BIGSERIAL,
+    completion_time BIGINT NOT NULL UNIQUE,
+    dv_pairs JSONB NOT NULL,
```

Comment:
Can we add some description for short formed column?

Reply:
Sure.